daemon/kmd/config/config.go

```diff
@@ -95,8 +96,15 @@ func LoadKMDConfig(dataDir string) (cfg KMDConfig, err error) {
 	cfg = defaultConfig(dataDir)
 	configFilename := filepath.Join(dataDir, kmdConfigFilename)
 	dat, err := ioutil.ReadFile(configFilename)
-	// Return the default configuration if the file doesn't exist
+	// If there is no config file, then return the default configuration, and dump the default config to disk
 	if err != nil {
+		exampleFilename := filepath.Join(dataDir, kmdConfigExampleFilename)
```

Comment:
Use codes.SaveObjectToFile() and save it formatted.

Reply:
Where to modify the default configuration ip and port?